Error while syncing engine config for flow test runner

I have the following engine configuration:
```
{
  "type": "flogo:engine",
  "imports": [
    "github.com/project-flogo/services/flow-state/store/mem@master"
  ],
  "actionSettings": {
    "github.com/project-flogo/flow": {
      "stepRecordingMode": "full",
      "snapshotRecordingMode": "on",
      "enableExternalFlows": true
    }
  },
  "services": [
    {
      "name": "flowTester",
      "ref": "github.com/project-flogo/flow/tester",
      "enabled": true,
      "settings": {
        "host": "localhost",
        "port": "8181"
      }
    },
    {
      "name": "flowStateRecorder",
      "ref": "github.com/project-flogo/services/flow-state/client/local",
      "enabled": true,
      "settings": {
      }
    },
    {
      "name": "flowStateProvider",
      "ref": "github.com/project-flogo/services/flow-state/server/rest",
      "enabled": true,
      "settings": {
        "host": "localhost",
        "port": "8080"
      }
    }
  ]
}
```

When trying to build my project with `flogo build --sync` I get the following error:
```
C:\Users\apandura\Downloads\Testing\MyTestApp\anand>flogo  build --sync
Error in installing 'github.com/project-flogo/services/flow-state/store/mem master'
Error synchronzing imports: go: finding github.com master
go: finding github.com/project-flogo/services/flow-state/store master
go: finding github.com/project-flogo/services/flow-state/store/mem master
go: finding github.com/project-flogo/services master
go: finding github.com/project-flogo/services/flow-state master
go: finding github.com/project-flogo master
# github.com/project-flogo/services/flow-state/store/mem
..\..\..\..\..\Desktop\AnandDocuments\gotest\pkg\mod\github.com\project-flogo\services\flow-state@v0.0.0-20191104212716-308c224cd5d0\store\mem\step.go:39:69: sc.f undefined (type *stepContainer has no field or method f)
```

---------------------------------------

I tried with `github.com/project-flogo/services/flow-state/store/mem` under my engine imports. I get the following error:
```
C:\Users\apandura\Downloads\Testing\MyTestApp\anand>flogo  build --sync
Error in installing 'github.com/project-flogo/services/flow-state/store/mem'
Error synchronzing imports: go: finding github.com/project-flogo/services latest
go: finding github.com/project-flogo/services/flow-state latest
# github.com/project-flogo/services/flow-state/store/mem
..\..\..\..\..\Desktop\AnandDocuments\gotest\pkg\mod\github.com\project-flogo\services\flow-state@v0.0.0-20191104212716-308c224cd5d0\store\mem\step.go:39:69: sc.f undefined (type *stepContainer has no field or method f)
```

--- a/.gitignore
+++ b/.gitignore
@@ -1,0 +1,49 @@
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
+
+## OS X Resources
+.DS_Store
+
+bin/
+pkg/
+**/vendor/src/
+**/vendor/manifest
+
+# Binaries
+*.exe
+*.test
+*.prof
+
+# Compiled Object files, Static and Dynamic libs (Shared Objects)
+*.o
+*.a
+*.so
+
+# Dynamic libs
+*.dll
+*.so
+*.dylib
+
+# Folders
+_obj
+_test
+
+# Architecture specific extensions/prefixes
+*.[568vq]
+[568vq].out
+
+*.cgo1.go
+*.cgo2.c
+_cgo_defun.c
+_cgo_gotypes.go
+_cgo_export.*
+
+_testmain.go
+
+.vscode/
+tags
+.vscode/symbols.json
+.idea
+.build-cache
+submodules/flogo-cicd/.build-cache
+./Dockerfile

--- a/flow-state/store/mem/step.go
+++ b/flow-state/store/mem/step.go
@@ -36,7 +36,7 @@ func (s *StepStore) GetFlow(flowId string) *state.FlowInfo {
 	s.RUnlock()
 
 	if ok {
-		return &state.FlowInfo{Id: flowId, Status: sc.Status(), FlowURI:sc.f}
+		return &state.FlowInfo{Id: flowId, Status: sc.Status(), FlowURI:sc.flowURI}
 	}
 
 	return nil

